src/prefect/cli/flow_run.py

```diff
@@ -196,9 +203,19 @@ async def logs(
             )
 
             # Get the next page of logs
-            page_logs = await client.read_logs(
-                log_filter=log_filter, limit=num_logs_to_return_from_page, offset=offset
-            )
+            if reverse:
+                page_logs = await client.read_logs(
+                    log_filter=log_filter,
+                    limit=num_logs_to_return_from_page,
+                    offset=offset,
+                    sort=schemas.sorting.LogSort.TIMESTAMP_DESC,
+                )
+            else:
+                page_logs = await client.read_logs(
+                    log_filter=log_filter,
+                    limit=num_logs_to_return_from_page,
+                    offset=offset,
+                )
```

Comment:
We could probably do this in a single line if statement, e.g.
```suggestion
            page_logs = await client.read_logs(
                log_filter=log_filter,
                limit=num_logs_to_return_from_page,
                offset=offset,
                sort=schemas.sorting.LogSort.TIMESTAMP_DESC if reverse else schemas.sorting.LogSort.TIMESTAMP_ASC
            )
```

Reply:
np thx for your review